Description: Cope with the ironical disappearance of the ISSUES file
 from the upstream tarball.
Author: Didier Raboud <odyx@debian.org>
Origin: vendor
Last-Update: 2012-07-10
--- a/setup.py
+++ b/setup.py
@@ -28,8 +28,7 @@
     url='http://github.com/vitorbaptista/pyppd/',
     license='MIT',
     description='A CUPS PostScript Printer Driver\'s compressor and generator',
-    long_description=open('README', 'rb').read().decode('UTF-8') + "\n" +
-                     open('ISSUES', 'rb').read().decode('UTF-8'),
+    long_description=open('README', 'rb').read().decode('UTF-8'),
     cmdclass={'sdist': sdist},
     classifiers=[
         'Development Status :: 5 - Production/Stable',
